### PR TITLE
Change default color to Mozilla accents

### PR DESCRIPTION
**Pull request by aaga**
Adds dark purple (for light mode) and light purple (for dark mode) as the default colors for our Elk instance.

Also fixes a bug where Dark Mode wasn't actually using the Dark Mode colors - this can be upstreamed.

### CODE REVIEW

**Comment by aaga** [comment]
Note that Moiz and I decided to leave the color button in the Settings menu as the darker purple even in Dark Mode, since the lighter purple is too close to the other purple in the theme

File: scripts/generate-themes.ts [hunk 1]
```diff
@@ -24,8 +24,9 @@ export function getThemeColors(primary: string): ThemeColors {
     '--c-dark-primary-active': dc.darken(0.5).hex(),
     '--c-dark-primary-light': dc.alpha(0.5).hex(),
     '--c-dark-primary-fade': dc.darken(0.1).alpha(0.1).hex(),
-    '--rgb-dark-primary': c.rgb().join(', '),
+    '--rgb-dark-primary': dc.rgb().join(', '),
```
Comment (jpezninjo) [inline scripts/generate-themes.ts:27]: was this the bug from Elk?

**Review by wtfluckey — APPROVED** [review]
YAY!

**Review by jpezninjo — APPROVED** [review]

File: scripts/generate-themes.ts (hunk 1 above)
Comment (aaga) [reply scripts/generate-themes.ts:27]: yep that was one part of it; the other was the CSS thing. I made it a separate commit so you can see it more clearly

**Pull request merged**